mava/systems/q_learning/rec_iql.py

```diff
@@ -599,7 +602,10 @@ def run_experiment(cfg: DictConfig) -> float:
         jax.block_until_ready(eval_output)
 
         # Log:
+        elapsed_time = time.time() - start_time
         episode_return = jnp.mean(eval_output.episode_metrics["episode_return"])
+        steps_per_eval = int(jnp.sum(eval_output.episode_metrics["episode_length"]))
```

Comment:
No need to make this an int as you're diving with it on the next line. Unless it's to get rid of the `Array` around it? In which case you can leave it

```suggestion
        steps_per_eval = jnp.sum(eval_output.episode_metrics["episode_length"])
```

Reply:
I followed the PPO logging method as closely as possible, which does use the `int()` here to get rid of the `Array` around `steps_per_second`.